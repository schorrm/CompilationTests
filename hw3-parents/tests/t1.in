void printOk(bool isOk) {
    if (isOk) printi(1);
}

void main()
{
    int x = 4;
    int y = 100 ;
    printOk(x>y);
    int z;
}