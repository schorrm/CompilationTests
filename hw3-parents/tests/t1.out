---end scope---
---end scope---
isOk BOOL -1
---end scope---
x INT 0
y INT 1
z INT 2
---end scope---
printi (INT)->VOID 0
printOk (BOOL)->VOID 0
main ()->VOID 0
